spec/requests/groups_controller_spec.rb

```diff
@@ -1053,6 +1053,29 @@ def expect_type_to_return_right_groups(type, expected_group_ids)
         expect(response.status).to eq(403)
       end
 
+      it "does not notify users when the param is not present" do
+        user2 = Fabricate(:user)
+
+        put "/groups/#{group.id}/members.json", params: { usernames: user2.username }
+
+        expect(response.status).to eq(200)
+
+        topic = Topic.find_by(title: "You have been added as a member of the #{group.name} group", archetype: "private_message")
```

Comment:
This seems like a precarious way to test this, since it depends on the english translation never changing.

One thing you could do is check to see if inviting the user increases the private message topic count or not.